src/components/Nav/Nav.scss

```
@@ -0,0 +1,149 @@
+@import '../../styles/variables.scss';
+
+$hover-color: #0042ff;
+
+.nav {
+  background-color: #fff;
+  min-width: 1080px;
+
+  .nav-top {
+    display: flex;
+    flex-direction: row-reverse;
+    height: 40px;
+    border-bottom: $border-line;
+  }
+
+  .sub-menu-list {
+    display: flex;
+    align-items: center;
+  }
+
+  .sub-menu-item {
+    margin: 0 10px;
+
+    .link-menu {
+      .ico-nav {
+        margin-right: 4px;
+        transition: background-image 0.2s;
+      }
+
+      .txt {
+        display: inline-block;
+        vertical-align: middle;
+        transition: color 0.2s;
+        font-size: 13px;
+      }
+
+      &:hover {
+        .ico-nav {
+          background-image: url(../../../public/images/nav/ico_nav_active.svg);
```

```suggestion
          background-image: url(/images/nav/ico_nav_active.svg);
```
public 폴더에 접근하고 할 때는 절대경로로 적어주세요! 
상대경로로 지정하게 되면 url path의 depth가 깊어졌을때 원하는대로 public 폴더에 접근을 하지 못합니다!

예시) 상대경로로 작성했을 시

- http://localhost:3000/main => 이경우에는 상대경로로 작성해도 public 폴더로 정상 인식해서 정상동작
- http://localhost:3000/detail/1 => 이경우에는 path의 depth가 깊어져서 상대경로로 지정하게되면 pubilc/detail 폴더안에서 찾으려고 하기 때문에 제대로 동작하지 않음.